app/controllers/admins/users_controller.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+module Admins
+  class UsersController < AdminsController
+    before_action :current_user, :super_admin?
+    before_action :authenticate_user!
+    before_action :set_user, only: %i[show edit update destroy]
+    before_action :super_admin?, only: %i[index edit update destroy]
+
+    def index
+      @users = User.all if super_admin?
+    end
+
+    def show; end
+
+    def new
+      @user = User.new
+      @path = '/admins/users'
```

Comment:
this is not good @path = '/admins/users'

@mmrshk

Reply:
I suggest making the path admin panel simply: /admin/{resources}